src/dscim/preprocessing/input_damages.py

```diff
@@ -656,36 +656,75 @@ def prep_mortality_damages(
     paths,
     vars,
     outpath,
+    mortality_version,
     path_econ="/shares/gcp/integration/float32/dscim_input_data/econvars/zarrs/integration-econ-bc39.zarr",
 ):
-    print(
-        "This function only works on mortality_v4 and mortality_v5 damages from the mortality repo's valuation. Earlier versions of mortality contain different variable definitions (per capita, not per capita, with or without histclim subtracted off."
-    )
-
     ec = EconVars(path_econ=path_econ)
 
     # longest-string gcm has to be processed first so the coordinate is the right str length
     gcms = sorted(gcms, key=len, reverse=True)
 
-    for i, gcm in enumerate(gcms):
-        print(gcm, i, "/", len(gcms))
+    if mortality_version == 0:
+        scaling_deaths = "epa_scaled"
+        scaling_costs = "epa_scaled"
+        valuation = "vly"
+    elif mortality_version == 1:
+        scaling_deaths = "epa_iso_scaled"
+        scaling_costs = "epa_scaled"
+        valuation = "vsl"
+    elif mortality_version == 4:
+        scaling_deaths = "epa_popavg"
+        scaling_costs = "epa_scaled"
+        valuation = "vsl"
+    elif mortality_version == 5:
+        scaling_deaths = "epa_row"
+        scaling_costs = "epa_scaled"
+        valuation = "vsl"
+    else:
+        raise ValueError("Mortality version not valid: ", str(mortality_version))
+
+    # We set 0 population to infinity so that per capita damages are 0 in locations with 0 population
```

Comment:
@JMGilbert @davidrzhdu based on convo today it sounds like we should implement this for other sector inputs if we don't already